vtexdocs/RapiDoc
# Feature/method path copy

#### What is the purpose of this pull request?

To add 'copy to clipboard' interactions to the documentation section that shows its corresponding path
![image](https://user-images.githubusercontent.com/39717968/172953737-6fbf70ac-0e2e-4716-8946-0488ea382dce.png)

[Further information and examples here.](https://www.notion.so/vtexhandbook/Adicionar-intera-o-copy-section-method-path-RapiDoc-7616e65b8e7540ee96631d759d67ae0d)

#### What problem is this solving?

The absence of the interactions to follow [the expected design](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=3696%3A238901) for the section showing the REST method and the path referring to the documentation accessed by the user.

#### How should this be manually tested?

It is strongly recommended that you read [this task documentation](https://www.notion.so/vtexhandbook/Adicionar-intera-o-copy-section-method-path-RapiDoc-7616e65b8e7540ee96631d759d67ae0d) to better understand the contribution code, also check if there are any typos or styling errors in it. As this contribution has been given a provisory version tag, you can view the achieved results on [this deploy preview](https://deploy-preview-46--elated-hoover-5c29bf.netlify.app/).

#### Screenshots or example usage

https://user-images.githubusercontent.com/39717968/172954914-bcf37e3d-9b10-481e-94e4-0b529e28ace1.mp4


#### Types of changes
